decentraland/explorer-desktop
# feat: av pro implementation

## What does this PR change?

Add the AVPro free plugin to be used as a video streaming framework.




## How to test the changes?

1) Open the desktop version using https://renderer-artifacts.decentraland.org/desktop/index.html?dcl://DESKTOP-BRANCH=feat/av-pro-implementation
2) Go to the following video links, and check that they are working correctly, both audio and video. All of the videos should have the RENDERHEADS watermark since we are using the free version of AVPro.
3) Test both in Windows and in Mac

- Filmrare (2,55)
Go to Root.AX. There are rooms with a video that shows different authors. No audio here
- Notus Finanse. (-22,37)
There is a CEO speaking just next to you
- DG Arena. (-118,120)
Go into the arena. There are different screens in the corners, as well as a poker table in the middle
- Wilderness. (-24,89)
There is a big screen with a tutorial on a corner of the scene. Look for a youtube icon, on top of that
- Art week. 
There are many experiences using video. Have a look around!
55,61
58,60
62,61
66,62
61,66
54,64
61,70
53,73
64,70
69,65
55,65 (Metaboralis, causes plenty of issues due to a large amount of videos. There is some problem handling in this scene)

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
